iceoryx_hoofs/test/moduletests/test_shared_memory.cpp

```diff
@@ -19,34 +19,53 @@
 
 #include "iceoryx_hoofs/internal/posix_wrapper/shared_memory_object/shared_memory.hpp"
 #include "iceoryx_hoofs/platform/stat.hpp"
+#include "iceoryx_hoofs/posix_wrapper/posix_call.hpp"
+
+#include <fcntl.h>
 
 namespace
 {
 using namespace testing;
+using namespace iox;
+using namespace iox::posix;
 
 class SharedMemory_Test : public Test
 {
   public:
     void SetUp() override
     {
-        internal::CaptureStderr();
+        testing::internal::CaptureStderr();
     }
 
     void TearDown() override
     {
-        std::string output = internal::GetCapturedStderr();
+        std::string output = testing::internal::GetCapturedStderr();
         if (Test::HasFailure())
         {
             std::cout << output << std::endl;
         }
     }
+
+    static constexpr const char SUT_SHM_NAME[] = "/ignatz";
+
+    iox::cxx::expected<iox::posix::SharedMemory, iox::posix::SharedMemoryError>
+    createSharedMemory(const iox::posix::Policy policy)
+    {
+        return iox::posix::SharedMemory::create(SUT_SHM_NAME,
+                                                iox::posix::AccessMode::READ_WRITE,
+                                                policy,
+                                                S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH,
+                                                128);
+    }
 };
 
+constexpr const char SharedMemory_Test::SUT_SHM_NAME[];
+
 TEST_F(SharedMemory_Test, CTorWithValidArguments)
 {
     auto sut = iox::posix::SharedMemory::create("/ignatz",
                                                 iox::posix::AccessMode::READ_WRITE,
-                                                iox::posix::OwnerShip::MINE,
+                                                iox::posix::Policy::PURGE_AND_CREATE,
                                                 S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH,
                                                 128);
```

Comment:
Since you created `createSharedMemory` wouldn't it make sense to use it here and in the following tests?